Example/Backpack/BPKThemeContainerController.h

```diff
@@ -23,9 +23,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 NS_SWIFT_NAME(ThemeContainerController) @interface BPKThemeContainerController : UINavigationController
 
-+ (BPKThemeContainerController *) themeContainerController;
-
-- (void) swapThemeContainers;
++ (void) swapThemeContainers;
```

Comment:
I decided to make this a class method, as it doesn't make much sense to force you to get the instance just to call an instance method on it 👍